docs/layout-of-source-files.rst

```diff
@@ -40,7 +40,7 @@ The comment is recognized by the compiler anywhere in the file at the
 file level, but it is recommended to put it at the top of the file.
 
 More information about how to use SPDX license identifiers
-can be found at the `SPDX website <https://spdx.org/ids-how>`_.
+can be found at the `SPDX website <https://spdx.dev/use/overview/>`_.
```

Comment:
Looking at the [last available snapshot](https://web.archive.org/web/20221203050047/https://spdx.dev/ids/#how), the equivalent of the old link would be https://spdx.dev/learn/handling-license-info/#how.

The new link probably leads to more detailed into, but the old one was nicer if you want to get straight to the point and just use it :)

Reply:
Neat, I even went to the examples hyperlink, but clearly didn't scroll far enough to see the actual examples.